.github/workflows/build.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
can we merge this PR first https://github.com/hyperjumptech/whatsapp-chatbot-connector/pull/4

then, this file need to be removed.